packages/compat/src/v1-addon.ts

```diff
@@ -374,23 +374,24 @@ export default class V1Addon implements V1Package {
   }
 
   private buildTreeForStyles(built: IntermediateBuild) {
+    let tree;
     if (this.customizes('treeForStyles')) {
-      todo(`${this.name} may have customized the app style tree`);
-    } else if (this.hasStockTree('styles')) {
-      // The typical way these get used is via css @import from the app's own
-      // CSS (or SCSS). There is no enforced namespacing but that is the
-      // common pattern as far as I can tell.
-      //
-      // TODO: detect people doing the right thing (namespacing with their own
-      // package name) and send them down the happy path. Their styles can
-      // just ship inside the package root and be importable at the same name
-      // as before. Detect people doing anything other than that and yell at
-      // them and set up a fallback.
-      built.trees.push(
-        this.transpile(this.stockTree('styles', {
+      // the user's tree returns their own styles with no "app/styles" wrapping
```

Comment:
@ef4 I believe https://github.com/embroider-build/embroider/issues/49 is fixed by this change, is that correct?

Reply:
This avoids running babel over the app styles, but we need to do the same for addon styles too.